web-animations-1/Overview.bs

```diff
@@ -2465,123 +2465,203 @@ Animation effects {#animation-effects}
-      on the endpoints of the [=active interval=].<br>
-      (a) An animation effect with no delay;
-      the [=start time=] and beginning of the [=active interval=]
-      are coincident.<br>
-      (b) An animation effect with a positive delay;
-      the beginning of the [=active interval=]
-      is deferred by the delay.<br>
-      (c) An animation effect with a negative delay;
-      the beginning of the [=active interval=]
-      is brought forward by the delay.
+      A comparison of [=local time=], [=active time=], and [=iteration time=]
+      for an animation with an [=iteration duration=] of 1s
+      and an [=iteration count=] of 2.5.
+      The [=animation/start time=] defined by the [=animation=]
+      and [=animation effect/end time=] defined by the [=animation effect=]
+      are also annotated.
     </figcaption>
   </figure>
 
-  An [=end delay=] can also be specified
-  but is primarily only of use when sequencing animations.
+  In addition to these time spaces
+  we can also refer to the <dfn noexport>document time space</dfn>,
+  which is time space of the [=time values=]
+  of the [=default document timeline=] of the {{Document}}
+  of the [=current global object=].
 
+  Note: While the time spaces themselves are not bounded,
+  Web Animations defines [=active time=] and the [=iteration progress=]
+  such that they are clamped to a set range as shown in the diagram.
+  For example, whilst a time of -1 second
+  is a valid time in <em>active time space</em>,
+  the procedure for calculating the [=active time=]
+  defined in [[#calculating-the-active-time]]
+  will never return a negative value.
 </div>
 
-  [=Animation effects=] define an <dfn>active interval</dfn>
+### The active interval, duration, and delays ### {#the-active-interval}
+
+  [=Animation effects=] define an <dfn>active interval</dfn>,
   which is the period of time during which the effect
-  is scheduled to produce its effect--
-  with the exception of [=fill modes=],
-  which apply outside the [=active interval=].
+  is scheduled to produce its effect
+  (excepting [=fill modes=],
+  which apply outside the [=active interval=]).
+  Each [=animation effect=] has only one such interval.
 
   The lower bound of the [=active interval=]
-  is defined by the [=start delay=].
-
-  The <dfn>start delay</dfn> of an [=animation effect=]
-  is a signed offset from the [=start time=] of the [=animation=]
-  with which the animation effect is associated.
+  by default corresponds to the [=start time=] of the [=associated animation=],
+  but can be shifted by the <dfn>start delay</dfn>,
+  which is a signed offset from the [=start time=] of the [=animation=].
 
-  The length of the [=active interval=] is called the [=active duration=],
+  The length of the [=active interval=] is called the <dfn>active duration</dfn>,
   the calculation of which is defined in [[#calculating-the-active-duration]].
+  Its length determines the upper bound of the [=active interval=].
```

Comment:
Perhaps we should just bring the the `#calculating-the-active-duration` section inline here?

As an implementer, when I go to the definition of "active duration", I want to see how to calculate (implement) it. Not its significance in the model.

Reply:
I definitely debated doing that. :) The main downside to it is that we haven't yet defined repetitions. But if you're OK with inlining the calculation here and pointing people forward for the definitions, I'm happy to do it. I agree it feels more natural.

Reply:
OK done. That prompted me to restructure the active interval section also. See the latest two new commits.

Reply:
Thanks! I left a comment on the first commit.